test/performance/benchmarks_test.go

```diff
@@ -87,10 +83,26 @@ func runTest(t *testing.T, pacer vegeta.Pacer, saveMetrics bool) {
 		t.Fatalf("Error probing domain %s: %v", domain, err)
 	}
 
+	endpoint, err := resolveEndpoint(clients.KubeClient, domain, test.ServingFlags.ResolvableDomain,
+		pkgTest.Flags.IngressEndpoint)
+	if err != nil {
+		t.Fatalf("Cannot resolve service endpoint: %v", err)
+	}
+
+	url := endpoint
```

Comment:
Why not name the variable "url" from the getgo?

Reply:
I concur, just use one variable.